discord/guild.py

```diff
@@ -1153,3 +1153,14 @@ def audit_logs(self, *, limit=100, before=None, after=None, reverse=None, user=N
 
         return AuditLogIterator(self, before=before, after=after, limit=limit,
                                 reverse=reverse, user_id=user, action_type=action)
+    @asyncio.coroutine
+    def webhooks(self):
+        """
+        Gets all guild webhooks
+
+        Returns:
+            list: 
```

Comment:
This documentation is completely wrong.

It should be:

```
Returns
---------
List[:class:`Webhook`]
    The webhooks for the guild.
```

Also missing the `Raises` section.